static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -260,9 +261,10 @@ export default function GroupSidebar({
 
   return (
     <Container>
-      {((organization.features.includes('gen-ai-features') &&
+      {((!organization.hideAiFeatures &&
         issueTypeConfig.issueSummary.enabled &&
-        !organization.hideAiFeatures) ||
+        organization.features.includes('gen-ai-features') &&
+        getRegionDataFromOrganization(organization)?.name !== 'de') ||
```

Comment:
Can we use `useAiConfig` here and use `areAiFeaturesAllowed` from there which should have the DE check? This will make sure we align all our AI conditionals and reduce copied logic

Reply:
Done